response_operations_ui/controllers/message_controllers.py

```diff
@@ -106,6 +106,34 @@ def remove_unread_label(message_id):
         logger.exception("Failed to remove unread label", message_id=message_id)
 
 
+def add_closed_conversation_label(thread_id):
+    url = f"{current_app.config['SECURE_MESSAGE_URL']}/v2/threads/{thread_id}"
+    data = {"is_closed": True}
+
+    logger.debug("Adding closed conversation label", thread_id=thread_id)
+    response = requests.patch(url, headers={"Authorization": _get_jwt(), "Content-Type": "application/json"}, json=data)
+
+    try:
+        response.raise_for_status()
+        logger.debug("Successfully added closed conversation label", thread_id=thread_id)
+    except HTTPError:
```

Comment:
I've raised an issue on other PRs around how we should be treating error codes differently (i.e. 4xx vs 5xx). I won't stop the PR on this but it's something to be aware of.

Reply:
Fair point, how should we be doing this? I'm guessing it's doing the same thing everywhere?

Reply:
Will be picked up in another piece of work by RAD team.